### PR TITLE
Add provenance to HTML output

### PR DESCRIPTION
**Pull request by benjeffery**
Screenshot from 2024-09-25 14-34-53

### CODE REVIEW

**Comment by jeromekelleher** [comment]
Awesome! What happens if there's a thousand of them? We'll have this in sc2ts

**Comment by jeromekelleher** [comment]
It's worth testing this on some recent sc2ts outputs to see what you get. They're available on Slack

**Comment by benjeffery** [comment]
Provenances are now summarised after the first 10:
Screenshot from 2024-09-26 12-05-13

**Review by jeromekelleher — COMMENTED** [review]
Looks great, but let's not bother summarising the rest

File: python/tskit/util.py [hunk 1]
```diff
@@ -514,21 +517,76 @@ def html_table(rows, *, header):
...
+                    </td>
+                </tr>
+            """
+        except Exception as e:
+            provenance_rows += (
+                f"""Could not parse provenance record: """
+                f"""{e.__class__.__name__} {str(e)}"""
+            )
+    extra_summary = collections.Counter()
+    for prov in extra_provenances:
```
[the remark this answers is not in the document]
Comment (benjeffery) [reply python/tskit/util.py]: Ok, removed that bit. Should be all covered now.

**Pull request merged**